### PR TITLE
Adds playlist list view trevor scandalios

### PR DESCRIPTION
**Pull request by scandycuz**
## What

- Adds playlist list view on small screens

### Also

- Adds transition animation to playlist grid items

## Test

1. Decrease your screen size to less than 600 pixels in width
2. Confirm that playlists are displayed in a list

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-2f70xgv Playlist list view](https://app.clickup.com/t/2f70xgv)

File: src/pages/home/playlists/PlaylistList.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      <Table>
+        <colgroup>
+          <col style={ { width: "15%" } } />
+        </colgroup>
+
+        <TableHead>
+          <MuiTableRow>
+            <TableHeadCell />
+            <TableHeadCell>Name</TableHeadCell>
+            <TableHeadCell>Songs</TableHeadCell>
```
Comment (j-mateo) [inline src/pages/home/playlists/PlaylistList.tsx:34]: I think these headers are redundant. The name of the playlist is implied, and the number of songs is appended with the word "songs"

Not blocking, but maybe something to discuss with design

**Review by j-mateo — APPROVED** [review]
lgtm! 💪🏽 
The list is definitely a lot more usable in smaller devices

File: src/pages/home/playlists/PlaylistList.tsx (hunk 1 above)
Comment (scandycuz) [reply src/pages/home/playlists/PlaylistList.tsx:34]: Sounds good, definitely happy to discuss any changes with design.

**Pull request merged**